lib/appium-for-mac.js

```diff
@@ -12,16 +12,18 @@ const REQ_A4M_APP_PATH = path.resolve('/Applications', 'AppiumForMac.app');
 const a4mLog = logger.getLogger('Appium4Mac');
 
 class AppiumForMac {
-  constructor () {
-    this.proxyHost = DEFAULT_A4M_HOST;
-    this.proxyPort = DEFAULT_A4M_PORT;
+  constructor (opts = {}) {
+    this.proxyHost = opts.a4mHost || DEFAULT_A4M_HOST;
+    this.proxyPort = opts.a4mPort || DEFAULT_A4M_PORT;
+    this.a4mAppPath = opts.a4mAppPath || REQ_A4M_APP_PATH;
+    this.killAllA4MAppBeforeStart = opts.killAllA4MAppBeforeStart || true;
     this.proc = null;
     this.jwproxy = new JWProxy({server: this.proxyHost, port: this.proxyPort});
   }
 
   async start () {
-    if (!await fs.exists(REQ_A4M_APP_PATH)) {
-      throw new Error('Could not verify AppiumForMacDriver install; please install to your /Applications folder');
+    if (!await fs.exists(this.a4mAppPath)) {
+      throw new Error(`Could not verify AppiumForMacDriver install in ${this.a4mAppPath}; please install it to ${this.a4mAppPath}`);
```

Comment:
it is a good practice to put variable values in quotes, so people could easily extract them if they contain spaces

Reply:
same below